providers/atlassian/connection/admin/connection.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Mondoo, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package admin
+
+import (
+	"errors"
+	"os"
+
+	"github.com/ctreminiom/go-atlassian/admin"
+	"go.mondoo.com/cnquery/v9/providers-sdk/v1/inventory"
+	"go.mondoo.com/cnquery/v9/providers/atlassian/connection/shared"
+)
+
+const (
+	Admin shared.ConnectionType = "admin"
+)
+
+type AdminConnection struct {
+	id     uint32
+	Conf   *inventory.Config
+	asset  *inventory.Asset
+	client *admin.Client
+	host   string
+}
+
+func NewConnection(id uint32, asset *inventory.Asset, conf *inventory.Config) (*AdminConnection, error) {
+	adminToken := conf.Options["admintoken"]
```

Comment:
this won't be set. Same applies to the other options you assume in the other connection types. You should define the flags that are needed for these and handle them

Reply:
They should be working now